### ISSUE TITLE
AnswerTest.testMultiMapService

### ISSUE THREAD

**Issue by tezc** [hazelcast/hazelcast#16011]
_maintenance_ (commit c0137b96e31a740e346d573112e0948dad6a79a6)

http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-3.maintenance-nightly/291/testReport/junit/com.hazelcast.test.starter.answer.test/AnswerTest/testMultiMapService/

**Stacktrace**

```
java.lang.UnsupportedOperationException: Method is not implemented in PartitionContainerAnswer: getMultiMapContainer
	at com.hazelcast.test.starter.answer.PartitionContainerAnswer.answer(PartitionContainerAnswer.java:48)
	at com.hazelcast.test.starter.answer.AbstractAnswer.answer(AbstractAnswer.java:98)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at com.hazelcast.multimap.impl.MultiMapPartitionContainer$MockitoMock$1646853771.getMultiMapContainer(Unknown Source)
	at com.hazelcast.test.starter.answer.test.AnswerTest.testMultiMapService(AnswerTest.java:426)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by tezc** [comment]
@blazember Is it related with https://github.com/hazelcast/hazelcast/commit/c0137b96e31a740e346d573112e0948dad6a79a6?

**Comment by blazember** [comment]
@tezc yes, I'm fixing this. Probably it needs a fix in 4.0 as well.

**Comment by blazember** [comment]
Closing as fixed by #16029 and no need to fix on 4.0.
